xlators/storage/posix/src/posix-helpers.c

```diff
@@ -2361,12 +2361,6 @@ posix_ctx_disk_thread_proc(void *data)
     };
 
     ctx = data;
-    interval = 5;
-
-    gf_msg_debug("glusterfs_ctx", 0,
-                 "Ctx disk-space thread started, "
-                 "interval = %d seconds",
-                 interval);
```

Comment:
Keep the log, just change the interval variable with the variable you added.

Reply:
@pranithk  i found  need pass xlator pointer to posix_ctx_disk_thread_proc,not glusterfs_ctx_t pointer.